scipy/stats/stats.py

```diff
@@ -3489,7 +3489,7 @@ def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate'):
     >>> tau
     -0.47140452079103173
     >>> p_value
-    0.24821309157521476
+    0.2827454599327748
```

Comment:
(Optional) Maybe add a very short (a couple of sentences) discussion of a possible interpretation of these numbers (inventing a meaning for the original data if necessary).